org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.target

```diff
@@ -1270,7 +1271,7 @@
 		<unit id="zweb" version="8.6.0.1"/>
 		<unit id="zweb.sources" version="8.6.0.1"/>
 		<unit id="org.apache.servicemix.bundles.xerces" version="2.9.1.5"/>
-		<repository location="https://raw.githubusercontent.com/idempiere/binary.file/master/p2.maven/maven-p2-R20190411"/>
+		<repository location="https://raw.githubusercontent.com/idempiere/binary.file/master/p2.maven/maven-p2-R20191224"/>
```

Comment:
mvn verify threw error:

[ERROR] Failed to resolve target definition /home/carlos/gitIdempiere/localdev/org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.target: Failed to load p2 metadata repository from location https://raw.githubusercontent.com/idempiere/binary.file/master/p2.maven/maven-p2-R20191224: No repository found at https://raw.githubusercontent.com/idempiere/binary.file/master/p2.maven/maven-p2-R20191224. -> [Help 1]

Reply:
Someone got to push maven-p2-R20191224 to https://github.com/idempiere/binary.file/tree/master/p2.maven first. Also, probably a good idea to make that a variable to make local testing/experiment easier.